src/components/ScholarshipListCard.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  Box,
+  Typography,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  actions: {
+    padding: theme.spacing(2),
+    paddingTop: theme.spacing(1),
+  },
+  content: {
+    padding: theme.spacing(3),
+  },
+  subtitle: {
+    fontWeight: '500',
+  },
+  deadline: {
+    fontWeight: 'bold',
+    marginBottom: theme.spacing(1),
+  },
+  description: {
+    display: '-webkit-box',
+    '-webkit-line-clamp': 5,
+    lineClamp: 5,
+    '-webkit-box-orient': 'vertical',
+    overflow: 'hidden',
+    whiteSpace: 'pre-line',
+    marginBottom: theme.spacing(3),
+  },
+  tagWrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+  tag: {
+    marginRight: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+    borderRadius: '50px',
+    border: '1px solid #3FB1B5',
+    padding: '12px 18px',
+  },
+}));
+
+function ScholarshipListCard({ scholarship, id }) {
+  const classes = useStyles();
+
+  return (
+    <Card variant="outlined">
+      <CardActionArea
+        component={Link}
+        to={{
+          pathname: `/scholarships/${id}`,
+          state: { scholarship: { id, scholarship } },
+        }}>
+        <CardContent className={classes.content}>
+          <Typography variant="body1" className={classes.deadline}>
+            {scholarship.deadline.toLocaleDateString()}
+          </Typography>
+          <Typography variant="h5">{scholarship.name}</Typography>
+          <Typography variant="subtitle1" className={classes.subtitle}>
+            {scholarship.city}
+          </Typography>
+          <Typography
+            variant="subtitle1"
+            className={classes.subtitle}
+            gutterBottom>
+            $1,000
```

Comment:
`{scholarship.amount.toString()}`

Reply:
After check i had found this comment will able to solve by changing ./src/testdata/scholarships.js.

changing list
- id from number to string,
- amount from number to object contains toString function

Reply:
Reason why we have to change this 
- data.id 
    itself generate by firestore so it should be string but now the testing data is number type
- data.amount 
    because in ScholarshipListCard need -> prop.amount.toString() <- for rendering so it much change from plain number to object contains toString function instead.

Reply:
Can i ?

Reply:
Ah, testdata/scholarships.js is old from before we had much structure in place. You can update that to match the expected [ScholarshipData interface](https://github.com/beyondhb1079/s4us/blob/main/src/models/Scholarships.ts#L6-L18)